docs/background/glossary.md

```diff
@@ -5,6 +5,9 @@
 Please note that this Glossary is **work in progress**. Do you encounter missing terms or want to suggest definitions, please let us know.
 
 - 3'rule: for all descriptions the most 3' position possible of the reference sequence is arbitrarily assigned to have been changed. When ATTTG changes to ATTG HGVS describes this as a change of the T at position 4 (not the T at position 2 or 3)
+
+- adjoined transcript: a transcript (RNA molecule) composed of adjoined RNA from two or more contributing transcripts.
```

Comment:
Up to you: Consider providing the definition, like this:

... `composed of <a href="https://www.google.com/search?q=adjoined+definition">adjoined</a> RNA` ...